### PR TITLE
add emulator_host

### PR DESCRIPTION
**Pull request by case-k-git**
set emulator_host to test in local env
https://cloud.google.com/pubsub/docs/emulator

https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-pubsub/lib/google/cloud/pubsub.rb#L96

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +101,8 @@ Use `gcloud_pubsub` output plugin.
   - Set Pub/Sub service endpoint. For more information, see [Service Endpoints](https://cloud.google.com/pubsub/docs/reference/service_apis_overview#service_endpoints)
 - `compression` (optional, default: `nil`)
   - If set to `gzip`, messages will be compressed with gzip.
+- `emulator_host`(optional)
+  - Host name of the emulator. Defaults to `ENV["PUBSUB_EMULATOR_HOST"]`, see [Testing apps locally with the emulator](https://cloud.google.com/pubsub/docs/emulator#accessing_environment_variables)
```
Comment (mia-0032) [inline README.md]: Is this default value `ENV["PUBSUB_EMULATOR_HOST"]` correct?

**Review by mia-0032 — COMMENTED** [review]
Could you rebase this branch? This PR conflicts with your last PR.

**Comment by Leodau** [comment]
This plugin should def be able to support using the local pub-sub emulator.

@mia-0032 Can I help?

**Comment by mia-0032** [comment]
@Leodau I think the `emulator_host` parameter is very useful. I hope you can either fix this PR or create a new one.

**Comment by case-k-git** [comment]
@mia-0032 
Sorry That  I did not fix this long time.. Could you please check it?

**Comment by mia-0032** [comment]
@case-k-git Thanks!

**Pull request merged**